scripts/ci/check-https-links.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
To be consistent with other script files, we can change to `#! /usr/bin/env bash`
```suggestion
#! /usr/bin/env bash

```